manifests/logging-stack/fluentbit-fluentBit.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     app.kubernetes.io/name: fluent-bit
 spec:
-  image: kubesphere/fluent-bit:v1.8.3
+  image: peter-fluentbit:v1.8.10
```

Comment:
V1.8.3 does not support this plugin, or there is a bug in v1.8.3? If true, we should build a fluent bit image of v1.8.10.

Reply:
Again, local testing.
I'm not sure there is an official docker image with the necessary plugins installed. 
One solution would be to build and distribute one that includes all the plugins that the operator supports, or just documenting how to build and use your own?
I think I prefer the first solution, but that requires a new version to be built and released for kubesphere/fluent-bit (not a big issue, I can create a PR for that in the correct repo as well of course).

As a side-note, I see that the `kubesphere` reops are archived, should we perhaps use `fluent/fluent-bit` instead in upcoming releases? Changing the names of the CRDs is trickier of course..